pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/MailboxIdUtils.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.query.mailbox;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.pinot.query.runtime.operator.OpChainId;
+
+
+// TODO: De-couple mailbox id from query information
+public class MailboxIdUtils {
```

Comment:
It feels more Object Oriented to encapsulate this logic in a dedicated object `MailboxId` instead of putting it in Utils class.

```java
MailboxId id = MailboxId.builder()
    .withRequestId(requestId)
    // etc ..
    .build();
id.toOpChainId();
```

Reply:
put it this way: mailboxID(str) --> opChainID(str) is currently 1:1 mapping
this assumption doesn't hold going forward and thus we would rely on other methods to derive the opChainIDs associate (and most likely the responsibility will be shifted to which ever class that needs to derive this)